### PR TITLE
Remove warning, remove build step, make compatible with a default linter on projects without flow 

### PR DESCRIPTION
**Pull request by ewnd9**
I could make 3 separate pull requests if you want

### CODE REVIEW

**Comment by ewnd9** [comment]
@nmn ping

File: src/index.js [hunk 1]
```diff
@@ -54,16 +56,6 @@ module.exports =
 
             const flowConfig = find(file, '.flowconfig')
             if (!flowConfig) {
-              if (!that.lastConfigError[file] ||
```
Comment (nmn) [inline src/index.js:57]: I want to keep this.
I'm willing to change the frequency, but the errors are important.

**Review by nmn — DISMISSED** [review]
Please undo that one change!
And great work on everything else.

**Comment by ewnd9** [comment]
> Actually, I'll rethink the error situation.

Happy to hear that :tada: Existence of such error implies that every project should use `flow` which is subjective and inconvenient for the old projects

**Comment by ewnd9** [comment]
@nmn Forgot to mention, please, release a new version on [`atom.io`](https://atom.io/packages/autocomplete-flow) so everybody could use this changes

**Comment by nmn** [comment]
@ewnd9 Will do.

**Comment by lenovouser** [comment]
@nmn I think you forgot to release a new version 😄

**Comment by thornjad** [comment]
@nmn please release this to apm

**Pull request merged**